Classes/Command/EventStoreCommandController.php

```diff
@@ -0,0 +1,116 @@
+<?php
+namespace Neos\Cqrs\Command;
```

Comment:
Now we have the `CommandHandlerInterface` and `CommandInterface` on the same namespace as the CLI commands, but IMO that's a [separate issue to solve ](https://discuss.neos.io/t/deprecating-subpackage-concept/1415/8?u=bwaidelich)